### ISSUE TITLE
[Android] Not possible to map touch inputs.

### ISSUE THREAD

**Issue by demodanza** [libretro/RetroArch#11021]
## Description

[Touch inputs don't register when trying to bind hotkeys, like Ai service button, for example. Premade hotkeys like down+select to bring up the menu work ingame though. I can map external controller inputs, select for hotkey and L1 for Ai service, but nothing happens when  pressing the same buttons after with touch.]

### Expected behavior

[Be able to bind touch inputs or at least have a reference list of touch input names (recognized internally by retroarch) to bind manually in the cfg.]

### Actual behavior

[No input is registered (time runs out) or "mouse1" is recognized because of pressing to much, with is not useful ingame.]

### Steps to reproduce the bug

1. [Go to Settings>Input>hotkeys]
2. [Try to bind input using any touch overlay]

### Bisect Results

[Never tried before last week when testing the Ai service, but I'll assume always.]

### Version/Commit
You can find this information under Information/System Information

- RetroArch: [4c6b139]

### Environment information

- OS: [Android 8.1.0 MIUI 10.3.2]

**Comment by hizzlekizzle** [comment]
I don't believe this is easily changed/fixed, as the overlays have direct references to retropad/retrokeyboard inputs and hotkey functions. However, I haven't checked whether the AI service hotkey is something that can be assigned to an overlay action and, if not, that's probably something that could be corrected. You would still need to modify an overlay to include the functionality, but that's easier to do than making the overlays mappable.
